packages/@react-aria/searchfield/test/useSearchField.test.js

```diff
@@ -134,6 +134,7 @@ describe('useSearchField hook', () => {
         expect(ref.current.focus).toHaveBeenCalledTimes(1);
         // Verify that props.onClear is triggered as well with the same event
         expect(onClear).toHaveBeenCalledTimes(1);
+        expect(onClear).not.toHaveBeenCalledWith(mockEvent);
```

Comment:
can you assert that it was called with nothing?
expect(onClear).toHaveBeenCalledWith()

or maybe there's a matcher that means nothing?

Reply:
ooo, lemme try that, makes more sense